src/js/jsx/sections/style/UnsupportedEffectList.jsx

```diff
@@ -161,6 +163,8 @@ define(function (require, exports, module) {
                 );
             }
             
+            headlights.logEvent("effect", "create", "unsupported");
```

Comment:
Can I add a componentDidMount method and have the headlights call be the only this in this method?